src/coreclr/jit/codegenarm64.cpp

```diff
@@ -5431,9 +5431,12 @@ void CodeGen::genProfilingLeaveCallback(unsigned helper)
  * disassembler thinks the instructions as the same as we do.
  */
 
-// Uncomment "#define ALL_ARM64_EMITTER_UNIT_TESTS" to run all the unit tests here.
+// Uncomment "#define ALL_ARM64_EMITTER_UNIT_TESTS" and one or more of the categories to run all the unit tests here.
 // After adding a unit test, and verifying it works, put it under this #ifdef, so we don't see it run every time.
-//#define ALL_ARM64_EMITTER_UNIT_TESTS
+#define ALL_ARM64_EMITTER_UNIT_TESTS
+// #define ALL_ARM64_EMITTER_UNIT_TESTS_GENERAL
+// #define ALL_ARM64_EMITTER_UNIT_TESTS_ADVSIMD
+// #define ALL_ARM64_EMITTER_UNIT_TESTS_SVE
```

Comment:
Thanks for adding that categories; hopefully that helps.

Reply:
I almost put this in another PR.

Also, the `INS_brk` test fails to compile because it should be `INS_brk_windows` and `INS_brk_linux`.

It be good if we could use the cstool output to automatically have this tested and checked.